sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -148,7 +148,11 @@ def test_lda_partial_fit_dim_mismatch():
                                     learning_offset=5., total_samples=20,
                                     random_state=rng)
     lda.partial_fit(X_1)
-    with pytest.raises(ValueError, match=r"^The provided data has"):
+    match = (
```

Comment:
now that we have the common check, should we just remove this?

Reply:
I was thinking of this. Let me do that.